armi/reactor/converters/uniformMesh.py

```diff
@@ -867,6 +838,64 @@ def _mapStateFromReactorToOther(
         self._cachedReactorCoreParamData = {}
 
 
+class GammaUniformMeshConverter(NeutronicsUniformMeshConverter):
+    """
+    A uniform mesh converter that specifically maps gamma parameters.
+    """
```

Comment:
I think it would be extremely useful to outline in the docstring for this class as well as the `NeutronicsUniformMeshConverter` the conditions which determine which parameters are mapped in each direction. 

I foresee a case where a plugin is mistakenly calculating a parameter that is not mapped back to the source reactor, and then the user/developer would be left wondering why their updated value did not get reflected back on their original reactor.

Making it clear in the docstring that certain parameters are not mapped to/from would help to guard against that kind of situation.

Reply:
Somehow I missed several of these comments the first time I looked through.

I added a lot of detail to the base class `UniformMeshGeometryConverter` docstring and to the derived class docstrings. I could definitely see that being a really frustrating error to contend with, so I'll try to put a clear warning in the docstring of each converter.